about.html

```diff
@@ -33,7 +32,7 @@
         </ul>
           </div>
           <div class="desktop-nav">
-            <a href="home.html"><img src="./Features/images/project-logo.png" alt="logo" class="logo"></a>
+            <a href="index.html"><img src="./Features/images/project-logo.png" alt="logo" class="logo2"></a>
             <div class="nav-flex">
```

Comment:
- [ ] like the previous reviewer mentioned, kindly reduce the width of the main about page heading, make it bolder as well, so it matches the original design. 
- [ ] please, the text with white background below it in a rectangular box should have its width reduced as well as in the Behance design. Nice work.
yours
![Screenshot (1180)_LI](https://github.com/Bupilipili/First-Capstone-Project/assets/65298048/39227538-8b9c-4a64-8dd0-efa39f6bf720)
behance
![Screenshot (1181)](https://github.com/Bupilipili/First-Capstone-Project/assets/65298048/b9fd9bd9-240f-4ef8-be3b-6cd6ebf8014d)